hello-world-service/main.bal

```diff
@@ -0,0 +1,18 @@
+import ballerina/http;
+
+listener http:Listener httpDefaultListener = http:getDefaultListener();
+
+service / on httpDefaultListener {
+    function init() returns error? {
+        do {
+
+        } on fail error err {
+
+        }
+    }
+
+    resource function get greeting(string name = "world") returns string {
+        // Returns the greeting message as a string
```

Comment:
I think we can remove the comments as well.

Reply:
Intentionally kept the comments wherever possible, as these samples supposed to be as easy as possible for the users to understand (IINM we follow the same approach in Ballerina by examples as well). wdyt?